layers/meta-resin-raspberrypi/recipes-kernel/linux/linux-raspberrypi_4.4.bbappend

```diff
@@ -1,4 +1,4 @@
-FILESEXTRAPATHS_prepend := "${THISDIR}/linux-raspberrypi-4.1:"
+FILESEXTRAPATHS_prepend := "${THISDIR}/linux-raspberrypi-4.4:"
```

Comment:
It also has these lines:
# Overlay for the RPI 7" DSI display
KERNEL_DEVICETREE_append = " overlays/rpi-ft5406-overlay.dtb"